backend/docs/api/management_v2_deployments.yaml

```diff
@@ -571,6 +571,94 @@ paths:
         Lists all release update types.
       tags:
       - Management API
+  /artifacts/list:
+    get:
+      description: |
+        Returns a collection of all artifacts.
+      operationId: List Artifacts with pagination
+      parameters:
+      - description: | 
+          Artifact(s) exact name(s) or prefix filter. 
+          Add `*` to the end for prefix matching.
+        in: query
+        name: name
+        schema:
+          type: array
+          items:
+            type: string
```

Comment:
you know reading that, and assuming I know nothing, which is not far from what I do know, I have no idea how to pass an array of strings/prefixes in a query. example would be nice.